apps/client/src/layout/components/footer/footer.css

```diff
@@ -0,0 +1,34 @@
+/**Custom footer component CSS*/
+
+.dashboard-footer-container { 
+    z-index: 1002; 
+    position:sticky;
+    bottom:0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 8px 30px;
```

Comment:
use design token and inline styles for these pixels

Reply:
Resolved